jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerAlignmentTests.java

```diff
@@ -0,0 +1,72 @@
+package io.github.epam.bootstrap.tests.composite.section.spinner;
+
+import io.github.com.sections.spinner.Spinner;
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.spinnerAlignment;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+
+public class SpinnerAlignmentTests extends TestsInit {
+
+    private static final String spinnerBorderStyle = "spinner-border";
+
+    private static final String spinnerMarginStyle = "mb-3";
+    private static final String spinnerJustifyContentStyle = "justify-content-center";
+    private static final String spinnerAlignCenterStyle = "align-items-center";
+    private static final String spinnerClearfixStyle = "clearfix";
+    private static final String spinnerTextCenterStyle = "text-center";
+
+
+    @DataProvider
+    public Object[][] spinnerData() {
+        return new Object[][] {
+                {spinnerAlignment.spinnerLeft},
+                {spinnerAlignment.flexJustifySpinner},
+                {spinnerAlignment.flexAlignCenterSpinner},
+                {spinnerAlignment.clearfixSpinner},
+                {spinnerAlignment.textCenterSpinner},
+        };
+    }
+
+    @DataProvider
+    public Object[][] spinnerStyleData() {
+        return new Object[][] {
+                {spinnerAlignment.spinnerLeft, spinnerMarginStyle},
+                {spinnerAlignment.flexJustifySpinner, spinnerJustifyContentStyle},
+                {spinnerAlignment.flexAlignCenterSpinner, spinnerAlignCenterStyle},
+                {spinnerAlignment.clearfixSpinner, spinnerClearfixStyle},
+                {spinnerAlignment.textCenterSpinner, spinnerTextCenterStyle},
+        };
+    }
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test(dataProvider = "spinnerData")
+    public void isValidationTest(Spinner spinner) {
+        spinner.childs().get(0).highlight();
```

Comment:
we should use 1 instead of 0. we can wait for Igor's changes or override it to work with 1

Reply:
Done